yaml/eventrouter.yaml

```diff
@@ -71,7 +71,7 @@ spec:
     spec:
       containers:
         - name: kube-eventrouter
-          image: gcr.io/heptio-images/eventrouter:latest
+          image: ghcr.io/kube-logging/eventrouter:0.4.0
```

Comment:
we could use latest here, I would prefer not having to update the version per release

Reply:
completely understandable. fixed in 5fc8fac683b6e6064fa46d07e695c0933937af13